string contur_id
---
map_contur_msg/map_contur_msg[] conturs